pkg/agent/datamodel/types.go

```diff
@@ -847,6 +847,14 @@ func (p *Properties) IsVHDDistroForAllNodes() bool {
 	return true
 }
 
+func (config *NodeBootstrappingConfiguration) Is2204VHD() bool {
+	if config.AgentPoolProfile.Distro == AKSUbuntuContainerd2204 || config.AgentPoolProfile.Distro == AKSUbuntuContainerd2204Gen2 {
```

Comment:
Hi Anuj, could you add ' || config.AgentPoolProfile.Distro == AKSUbuntuArm64Containerd2204Gen2' to cover the ARM64 2204 distro. I added AKSUbuntuArm64Containerd2204Gen2 distro info to the Agentbaker repo last week.

Reply:
added a helper func that covers all 3 2204 distros